host_vars/dev.usegalaxy.org.au.yml

```diff
@@ -134,6 +134,17 @@ galaxy_jobconf:
             amqp_consumer_timeout: 2.0
             amqp_publish_retry: True
             amqp_publish_retry_max_retries: 60
+      - id: pulsar_eu_gpu
```

Comment:
Additions to this block will not have effect because the dev_job_conf.yml.j2 file in templates/galaxy/config is being used for the job conf.  These parts need to be added to dev_job_conf.yml. The syntax is slightly different